Shopipy.Web/Controllers/UserController.cs

```diff
@@ -0,0 +1,65 @@
+using Microsoft.AspNetCore.Mvc;
+using Shopipy.ApiService.DTOs;
+using Shopipy.Web.DTOs;
+using Shopipy.Web.Services;
+using System.Text;
+using System.Text.Json;
+
+namespace Shopipy.Web.Controllers;
+
+public class UserController(UserService userService, ILogger<UserController> logger, IHttpClientFactory httpClientFactory) : BaseController(userService)
+{
+
+    [HttpGet]
+    public IActionResult Login()
+    {
+        if (User.Identity?.IsAuthenticated == true)
```

Comment:
```suggestion
        if (User.Identity?.IsAuthenticated)
```

Reply:
cannot implicitly convert type system.nullable<bool> to bool